README.md

```diff
@@ -213,26 +213,29 @@ Short backend is built on top of
 [Uncle Bob's Clean Architecture](https://api.short-d.com/r/ca), the central
 objective of which is separation of concerns.
 
-![Clean Architecture](doc/eng/clean-architecture/clean-architecture.jpg)
-![Boundary](doc/eng/clean-architecture/boundary.jpg)
+![Clean Architecture](doc/clean-architecture/clean-architecture.jpg)
+
+![Short Backend](doc/clean-architecture/short-backend.jpg)
```

Comment:
In the image, is it should be Use Cases instead of User Cases.

Reply:
Good catch!

Reply:
Fix 2 diagrams.